src/getHumanAge.test.js

```diff
@@ -3,8 +3,23 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
+  test('getHumanAge is a function', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('age of animals is less than 15', () => {
```

Comment:
reconsider the tests title, at this moment it will shown in the next way:
getHumanAge age of animals is less than 15
(The test case summary consists of describe title + test title)
So in this case you should rewrite the summary something like: should return [0, 0] when animals' ages are less than 15